tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -8,24 +8,21 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 Connect container to a new network
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create test-network
     Should Be Equal As Integers  ${rc}  0
-
-# TEST TEMPORARILY DISABLED
-
-#    ${status}=  Get State Of Github Issue  1235
-#    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-17-Docker-Network-Connect.robot needs to be updated now that Issue #1235 has been resolved
-#    Log  Issue \#1235 is blocking implementation  WARN
-#    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} pull busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create busybox ifconfig
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect test-network ${containerID}
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} logs ${containerID}
-#    Should Be Equal As Integers  ${rc}  0
-#    Should Contain  ${output}  eth0
-#    Should Contain  ${output}  eth1
+    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create busybox ifconfig
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect test-network ${containerID}
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
+    Should Be Equal As Integers  ${rc}  0
+    ${status}=  Get State Of Github Issue  366
```

Comment:
Issue 366 is only preventing the final verification from working, the rest of the test runs fine and at least verifies that we don't return error codes.  But the final verification is needed to verify that we actually added a second network to within the container.